src/Worker.Extensions.DurableTask/DurableTaskClientExtensions.cs

```diff
@@ -120,8 +120,37 @@ public static HttpResponseData CreateCheckStatusResponse(
         return response;
     }
 
+    /// <summary>
+    /// Creates an HTTP management payload for the specified orchestration instance.
+    /// </summary>
+    /// <param name="client">The <see cref="DurableTaskClient"/>.</param>
+    /// <param name="instanceId">The ID of the orchestration instance.</param>
+    /// <param name="request">Optional HTTP request data to use for creating the base URL.</param>
+    /// <returns>An object containing instance control URLs.</returns>
+    /// <exception cref="ArgumentException">Thrown when instanceId is null or empty.</exception>
+    /// <exception cref="InvalidOperationException">Thrown when a valid base URL cannot be determined.</exception>
+    public static object CreateHttpManagementPayload(
+        this DurableTaskClient client,
+        string instanceId,
+        HttpRequestData? request = null)
+    {
+        if (string.IsNullOrEmpty(instanceId))
+        {
+            throw new ArgumentException("InstanceId cannot be null or empty.", nameof(instanceId));
+        }
+
+        try
+        {
+            return SetHeadersAndGetPayload(client, request, null, instanceId);
+        }
+        catch (InvalidOperationException ex)
+        {
+            throw new InvalidOperationException("Failed to create HTTP management payload. " + ex.Message, ex);
+        }
```

Comment:
Instead of catching and re-throwing the InvalidOperationException. Why don't we just throw a single exception inside of `SetHeadersAndGetPayload` and let that exception propagate out of this method? We may need to move the "Failed to create HTTP management payload" message to "SetHeadersAndGetPayload", but that seems 'ok' to me.